bzt/engine/engine.py

```diff
@@ -705,6 +706,8 @@ def eval_env(self):
         """
         envs = self.config.get(SETTINGS, force_set=True).get("env", force_set=True)
         envs[TAURUS_ARTIFACTS_DIR] = self.artifacts_dir
+        envs[TAURUS_USER_CONFIGS] = ','.join(map(str, self.config.get('user-configs', force_set=True)))
```

Comment:
could you explain the reason to use force_set here?

Reply:
Not valid question now